src/ruby/puppetserver-lib/puppet/server/catalog.rb

```diff
@@ -0,0 +1,121 @@
+require 'puppet/server'
+
+module Puppet::Server::Catalog
```

Comment:
I usually do
```rb
module Puppet
  module Server
    module Catalog
    end
  end
end
```
Is that unnecessary here?

Reply:
Oh.. yeah there are few weird edge cases, see https://techblog.thescore.com/2014/05/28/how-you-nest-modules-matters-in-ruby/. The biggest problem only arises if `Puppet::Server` were not defined in its own right (though both parts are in this case, so it works). This does work for me, but I'm happy to switch it.

Reply:
Confirmed that both work in this case.